recipe/meta.yaml

```diff
@@ -1,41 +1,41 @@
 {% set name = "dask-sql" %}
-{% set version = "2022.8.0" %}
+{% set version = "2022.9.0.rc0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_sql-{{ version }}.tar.gz
-  sha256: 0452c174c3473768988da55a78541b192264bd8a9e6090e9e7167d3e99a1ceec
+  url: https://github.com/dask-contrib/dask-sql/archive/{{ version }}.tar.gz
+  sha256: 8cf7280cb6529603cb11526932c7485effc6f4efeef22a41a278609bf0a91c9e
 
 build:
+  skip: true   # [py2k]
   entry_points:
     - dask-sql-server = dask_sql.server.app:main
     - dask-sql = dask_sql.cmd:main
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   build:
-    - maven >=3.6.0
+    - {{ compiler('rust') }}
+    - setuptools-rust >=1.4.1
   host:
     - pip
-    - python >=3.8
+    - python
```

Comment:
might need to add a min pin back to 3.8

Reply:
IIRC, this causes issues in conjunction with the conda build config file - think the issue here was that I forgot to commit the config specifying what Python versions to build for

Reply:
My bad, you were right - looks like conda-forge handles its build version matrix differently from the nightlies.

I've made the suggested modifications and re-rendered, so things should be working now